core-bundle/src/Image/Studio/Figure.php

```diff
@@ -203,11 +202,10 @@ public function getOptions(): array
      *       when using Twig templates! Instead, add this object to your
      *       template's context and directly access the specific data you need.
      *
-     * @param string|array|null $margin              Set margins that will compose the inline CSS for the "margin" key
-     * @param string|null       $floating            Set/determine values for the "float_class" and "addBefore" keys
-     * @param bool              $includeFullMetadata Make all metadata available in the first dimension of the returned data set (key-value pairs)
+     * @param string|null $floating            Set/determine values for the "float_class" and "addBefore" keys
+     * @param bool        $includeFullMetadata Make all metadata available in the first dimension of the returned data set (key-value pairs)
      */
-    public function getLegacyTemplateData(array|string|null $margin = null, string $floating = null, bool $includeFullMetadata = true): array
+    public function getLegacyTemplateData(string $floating = null, bool $includeFullMetadata = true): array
```

Comment:
As these methods (getLegacyTemplateData and applyLegacyTemplateData) are getting deprecated in the long term anyways, I’d prefer to keep this parameter to keep the method signature the same. With a comment maybe like `Deprecated parameter, has no effect anymore`

@m-vo WDYT?

Reply:
Yes, I think so, too. :+1:

Reply:
Fixed in 349741f